### DISCUSSION TITLE
Issue on using multiple nodes on HPC

### DISCUSSION THREAD

**Discussion by dineshkumarreddyg36** [xcompact3d/Incompact3d#150] [Q&A] [answered]
I'm trying to run the two_turbine default case that is given in the solver on HPC for which I'm using 2 nodes (20 cores per node). The simulation is getting hanged (neither throwing an error nor stopping) after some time steps. But if I run it on 1 node it's running fine. Could someone please comment on this?
Thank you.

**Comment by slaizet** [comment]
Hi, 
We just updated the master branch of the code to fix a small bug that might be related to your problem. I have tried the two_turbine default case on multiple nodes and on various supercomputers and did not encounter any issues with the fix.
Best,
Sylvain

**Reply by dineshkumarreddyg36** [reply]
But, on compiling the new code I'm encountering the following error and I've commented those lines that are causing the error and it compiled successfully. Still facing the same problem. The simulation is getting hanged but this time running for a little more number of iterations compared to the old code. Please look into it.
Thank you.
![error](https://user-images.githubusercontent.com/106377425/221097942-9d86bcb0-1b75-459d-9ea5-be1de6684978.JPG)

**Reply by pbartholomew08** [reply]
`is_contiguous` is a Fortran 2008 intrinsic. Using a more recent version of the compiler should resolve this.

**Reply by dineshkumarreddyg36** [reply]
I'm using gcc 7.2

**Comment by slaizet** [comment, marked as answer]
gcc 7.2 is from 2017 so you might want to update your compiler to a more recent version (it is very likely why you cannot compile)
The current master branch has been tested with various recent gcc versions and it is compiling fine.
Best,
Sylvain
